js/controllers/modal/ModalHandler.js

```diff
@@ -47,7 +47,7 @@
 				titleIcon: string }} */ (this.mergeOptions(internalOptions));
 
 		// Attach content to the modal
-		$handledElement.html(this.modalBuild().html());
+		$handledElement.html(this.modalBuild()[0].outerHTML);
```

Comment:
Sorry, my bug -- this was in response to a JS linting error but I didn't test the change.